src/components/dashboard/subscriptions/SubscriptionListContainer.js

```diff
@@ -0,0 +1,84 @@
+import React, {Component, PropTypes} from 'react';
```

Comment:
I noticed we weren't breaking lists into containers, good to refactor/use containers for them - nice!